spiceaidocs/docs/cli/reference/init.md

```diff
@@ -6,18 +6,20 @@ pagination_next: null
 ---
 Initialize Spice app - initializes a new Spice app
 
-### Usage:
+### Usage
+
 ```shell
 spice init [flags]
 ```
 
-#### Flags:
-  - `-h`, `--help`   Print this help message
+#### Flags
+
+- `-h`, `--help`   Print this help message
```

Comment:
Propose to remove  "Print this help message" from all pages as it confuses IMO 

>- `-h`, `--help` Print this help message

Or make it command specific - `help for init command`, etc